src/viking.js

```diff
@@ -1,11 +1,66 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack (){
+        return this.strength;
+    }
+    receiveDamage(damage){
+        this.health = this.health - damage
+    } 
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength){
+        super(health, strength);
+        this.name = name;
+        this.health = health;
+        this.strength = strength;
+    }
```

Comment:
aqui no hace falta volver a definir `this.health` y `this.strength,` ya que los estás recibiendo de la clase Soldier con `super(health, strength)`